<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addRowDialog</class>
 <widget class="QDialog" name="addRowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Hotel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nameLine"/>
   </item>
   <item>
    <widget class="QLabel" name="placeLabel">
     <property name="text">
      <string>Place:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="placeLine"/>
   </item>
   <item>
    <widget class="QLabel" name="typeLabel">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="typeLine"/>
   </item>
   <item>
    <widget class="QLabel" name="priceLabel">
     <property name="text">
      <string>Price:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="priceBox">
     <property name="maximum">
      <number>60000</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="starsLabel">
     <property name="text">
      <string>Stars:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="starsBox">
     <property name="maximum">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="ratingLabel">
     <property name="text">
      <string>Rating:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="ratingBox"/>
   </item>
   <item>
    <widget class="QLabel" name="cityLabel">
     <property name="text">
      <string>City:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="cityLine"/>
   </item>
   <item>
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
